example/image-classification/common/fit.py

```diff
@@ -150,6 +156,17 @@ def fit(args, network, data_loader, **kwargs):
     if args.gc_type != 'none':
         kv.set_gradient_compression({'type': args.gc_type,
                                      'threshold': args.gc_threshold})
+    if args.profile_server_suffix:
+        mx.profiler.set_config(filename=args.profile_server_suffix, profile_all=True, profile_process='server')
+        mx.profiler.set_state(state='run', profile_process='server')
+
+    if args.profile_worker_suffix:
+        if kv.num_workers > 1:
+            filename = 'rank' + str(kv.rank) + '_' + args.profile_worker_suffix
+        else:
+            filename = args.profile_worker_suffix
+        mx.profiler.set_config(filename=filename, profile_all=True, profile_process='worker')
```

Comment:
why necessary to pass profile_process to both with the same value? any way not to? seems tedious

Reply:
hmm, for worker it is not necessary, as the default value for profile_process would be worker. But for server, I can't think of a way to avoid it unless I create different wrapper functions for server. 
Or maybe create a class for a 'profile', and call set_config, and set_state on that object. 
I mean the following
```
prof = mx.profiler.job(profile_process='server')
prof.set_config
prof.set_state
```
Ignore the word job, we can come up with a better name if we want to go this route. But it feels unnecessary to me.